Sprintfinity3902/States/ItemPickup/HeartPickup.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Sprintfinity3902.Link;
+using Sprintfinity3902.Interfaces;
+
+namespace Sprintfinity3902.Entities
+{
+    public class HeartPickup : IPickup
+    {
+        public HeartPickup()
+        { 
+
+        }
+
+        public bool Pickup(Player Link)
+        {
+            if (Link.LinkHealth < Link.MaxHealth)
+            {
+                Link.LinkHealth++;
+                if (Link.LinkHealth != Link.MaxHealth)
+                {
+                    Link.LinkHealth++;
+                }
+                Link.heartChanged = true;
+                Sound.SoundLoader.Instance.GetSound(Sound.SoundLoader.Sounds.LOZ_Get_Heart).Play(Global.Var.VOLUME, Global.Var.PITCH, Global.Var.PAN);
```

Comment:
Heart pickup sound is not working at runtime for me. Are you having the same issue?

Reply:
Yup, this is a bug. Heart sound only plays if link is damaged.

Updating so it plays on every pickup.